.github/workflows/lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Run lint and static analyis checks
+on:
+  pull_request:
+  workflow_call:
+
+jobs:
+  tardigradelint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone this git repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

Comment:
See comment below that the name is the same but the code is different.

Reply:
fixed